pkg/providers/amazon/cloudformation/cloudformation_test.go

```diff
@@ -0,0 +1,196 @@
+// Copyright © 2018 Banzai Cloud
```

Comment:
```suggestion
// Copyright © 2020 Banzai Cloud
```
:)

Reply:
Nice catch.

Reply:
I have it on my todo list to write a script for these updates and run it on the repos.

Reply:
[Fixed](https://github.com/banzaicloud/pipeline/compare/05ede4fc01de703f51a210c39e39351aa6c7cfed..16dae79929e53106941ef86ba36540c57a57c320).

Reply:
(If you do such a script, it should use the git log, and list all the years when changes were made. You can't just claim new copyright on content authored earlier.)

Reply:
On my last project we used the `originalYear-CurrentYear` format and we always updated the current year with the script so it was determinable the copyright claim started at the point of origin and it was continuous to present day (we updated the values in January). That's the format I planned to use.